src/features/rewards/tab/index.tsx

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledSlider,
+  StyledBullet,
+  StyledTextOne,
+  StyledTextTwo,
+  StyledSwitch
+} from './style'
+
+export interface Props {
+  id?: string
+  tabOneChecked?: boolean
+  tabOne?: string
+  tabTwo?: string
+  onSwitch?: (event: React.MouseEvent<HTMLDivElement>) => void
+}
+
+export default class Tab extends React.PureComponent<Props, {}> {
+
+  render () {
+    const {
+      id,
+      tabOne,
+      tabTwo,
+      onSwitch,
+      tabOneChecked
+    } = this.props
+
+    return (
+      <StyledWrapper id={id}>
```

Comment:
IMO let's start giving 'StyledWrapper' a more unique name - when inspecting the code almost everything is `StyledWrapper` class, and I'd like to see something indicating what component it is. In this case, we could call the outer element the `RewardsTabWrapper`

Reply:
done